.travis.yml

```diff
@@ -54,7 +54,6 @@ matrix:
 
     - os: linux
       dist: trusty
-      group: deprecated-2017Q2
```

Comment:
there appears to be a strange correlation between unpinning this and the presence of #4868 - but possible that it's just coincidental.

at any rate, would suggest deleting the travis cache prior to firing new CI jobs for this PR as that appears to have bit us prior.

Reply:
Did this and things seem ok.